website/content/docs/configuration/vault.mdx

```diff
@@ -153,30 +129,6 @@ agents with [`server.enabled`] set to `true`.
   [`name`](#name) parameter. Setting a default identity causes the value of
   `allow_unauthenticated` to be ignored.
 
-### Deprecated Parameters
-
-These parameters are used by the deprecated token-based authentication flow and
-will be removed in a future release.
-
-- `allow_unauthenticated` `(bool: true)` - Specifies if users submitting jobs to
-  the Nomad server should be required to provide their own Vault token, proving
-  they have access to the policies listed in the job. This option should be
-  disabled in an untrusted environment.
-
-- `task_token_ttl` `(string: "72h")` - Specifies the TTL of created tokens when
-  using a root token. This is specified using a label suffix like "30s" or "1h".
-
-- `token` `(string: "")` - Specifies the parent Vault token to use to derive
-  child tokens for jobs requesting tokens. Only required on Nomad servers.
-  Nomad client agents use the allocation's token when contacting Vault.
-  Refer to the [Vault Integration Guide](/nomad/docs/integrations/vault-integration)
-  for how to generate an appropriate token in Vault.
```

Comment:
I think we need to move this so that we have it for servers that are using KMS with Vault transit encryption. (But that's the only reason you need a Vault block _at all_ on servers now.)

Reply:
This lives in the [keyring configuration block](https://developer.hashicorp.com/nomad/docs/configuration/keyring/transit) rather than the Vault block, so I believe removing it from here is correct.

Reply:
LOL, I wrote that code too... you'd think I'd have remembered that. :blush:

Reply:
You mean you can't remember every single line of code you've ever written...filthy casual :p